src/components/BaseLayerToggler/BaseLayerToggler.js

```diff
@@ -330,43 +284,55 @@ class BaseLayerToggler extends Component {
 
     if (layers.length > 2) {
       footer = (
-        <Footer>
-          <Button
-            className={classNamePrevious}
+        <div className="rs-base-layer-footer">
+          <div
+            className="rs-base-layer-previous"
+            role="button"
             onClick={() => this.previous()}
-            title={titleButtonPrevious}
+            onKeyPress={e => e.which === 13 && this.previous()}
+            tabIndex="0"
+            title={titles.prevButton}
           >
             <FaArrowCircleLeft focusable={false} />
-          </Button>
-          <Button
-            className={classNameNext}
+          </div>
+          <div
+            className="rs-base-layer-next"
+            role="button"
             onClick={() => this.next()}
-            title={titleButtonNext}
+            onKeyPress={e => e.which === 13 && this.next()}
+            tabIndex="0"
+            title={titles.nextButton}
           >
             <FaArrowCircleRight focusable={false} />
-          </Button>
-        </Footer>
+          </div>
+        </div>
       );
     }
 
     const nextLayer = layers[idx];
 
     return (
       <div className={className} ref={this.ref}>
```

Comment:
Since className is a common html attribute. I've put the default value here (see other cleanup) and then we apply the other props .
```
const { titles, ...other } = this.props;
      <div className='rs-base-layer.toggler'  ref={this.ref} {...other} >
```
What do you think?

Reply:
The advantage of keeping it as a default prop would be, to have it appear in the documentation. If you just want to modify the default style with a custom className a little bit, you could add the default classname together with a custom className without looking into the code.